### PR TITLE
backport 4.3: hwreq for /var/spacewalk (#2448)

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/uyuni-project/uyuni-docs/issues/2446
https://github.com/SUSE/spacewalk/issues/22514
Co-authored-by: Ornela Marić <omaric@suse.com>
* Update hardware-requirements.adoc
* Update uyuni-install-requirements.adoc

# Target branches

Which documentation version does this PR apply to?
- [ ] master (Default)
- [x] manager-4.3

# Links
- Related development PR #<insert PR link, if any>
- Fixes #<insert spacewalk issue, if any>

### CODE REVIEW

File: modules/installation-and-upgrade/pages/hardware-requirements.adoc [hunk 1]
```diff
@@ -56,7 +56,11 @@ You can estimate how much space the [path]``/var/spacewalk/`` directory requires
 
 |
 | [path]``/var/spacewalk``
-| Minimum 50{nbsp}GB per {suse} product and 360{nbsp}GB per Red Hat product
+| Minimum 100{nbsp}GB (the implemented check ensure it)
```
Comment (jcayouette) [inline modules/installation-and-upgrade/pages/hardware-requirements.adoc:59]: Minimum storage required: 100 GB (this has been verified by our implemented check).

File: modules/installation-and-upgrade/pages/hardware-requirements.adoc [hunk 2]
```diff
@@ -56,7 +56,11 @@ You can estimate how much space the [path]``/var/spacewalk/`` directory requires
 
 |
 | [path]``/var/spacewalk``
-| Minimum 50{nbsp}GB per {suse} product and 360{nbsp}GB per Red Hat product
```
Comment (jcayouette) [inline modules/installation-and-upgrade/pages/hardware-requirements.adoc:59]: Minimum storage requirements: 50 GB for each SUSE product and 360 GB for each Red Hat product.

**Comment by keichwa** [comment]
I hope I got it right.  Maybe, you can check again.

File: .changelog [hunk 3]
```diff
@@ -8,6 +8,8 @@
   Adminsitration Guide
 - Updated autoinstallation chapter about
   buildiso command in the context of Cobbler in Client Configuration Guide
+- Update Hardware Requirements section about disk space for /var/spacewalk in
```
Comment (0rnela) [inline .changelog:11]:
```suggestion
- Update Hardware Requirements section about disk space for /var/spacewalk in the
```

**Pull request merged**